Sources/HipMobileUI/ViewModels/Pages/ExhibitDetailsViewModel.cs

```diff
@@ -191,22 +201,34 @@ private void ToggleVisibilityOfNavigationButtons ()
         /// Go to the next available view.
         /// </summary>
         /// <returns></returns>
+    
+
+
         private async Task GotoNextView ()
         {
             if (currentViewIndex < pages.Count - 1)
             {
                 // stop audio
                 if (AudioToolbar.AudioPlayer.IsPlaying)
                 {
-                    AudioToolbar.AudioPlayer.Stop ();
+                    AudioToolbar.AudioPlayer.Stop();
                 }
-
                 // update the UI
+            }
+            if (exhibitUnblocked)
```

Comment:
Judging by the context this if-else-statement should be inside the upper if (currentViewIndex..) code segment after the statement related to the AudioPlayer. Due to NextViewAvailable it should cause no problem if it stays like this but it would be logically cleaner.